.github/workflows/integration.yaml

```diff
@@ -475,3 +474,45 @@ jobs:
             wazero run -mount=/:/ -env PWD=$PWD $bin -- -test.short -test.v
             popd
           done
+
+      - if: ${{ runner.os == 'Windows' }}
+        name: Run standard library tests
+        # Ack failures on Windows. https://github.com/tetratelabs/wazero/issues/1410
+        continue-on-error: true
+        run: |
+          GOOS=$(go env GOOS)
+          echo "Running $(find ~/sdk/tests -name *.test | wc -l) test binaries"
+
+          GOROOT=~/sdk/gotip
+
+          MOUNT=c:\\:/
+          SCRIPT="$HOME/sdk/tests.cmd"
+          # Trim `/c` from the in-Wasm GOROOT.
+          GOROOT=$(cygpath -u $GOROOT | cut -c 3-)
+          # Append early exit on cmd.
+          POSTFIX="if %errorlevel% neq 0 exit /b %errorlevel%"
+          RUNNER="cmd //c %USERPROFILE%\sdk\tests.cmd"
+          EXTRAPARAMS="-mount=%TEMP%:/tmp"
+
+          # Go tests often look for files relative to the source. Change to the corresponding directory.
+          for bin in ~/sdk/tests/*.test; do
+            dir=$(basename $bin); dir=${dir%.test}; dir=${dir//_/\/}
+            pushd ~/sdk/gotip/$dir
+
+            # Trim `/c` from the in-Wasm pwd.
+            IN_WASM_PWD=$(pwd | cut -c 3-)
+            # Convert to a Windows path.
+            bin=`cygpath -w $bin`
+
+            # Create a script with all the tests (do not run yet).
+            echo ${MOUNT} ${IN_WASM_PWD} ~/sdk/gotip/$dir
+            COMMAND="wazero run -mount=${MOUNT} ${EXTRAPARAMS} -hostlogging=filesystem -env PWD=${IN_WASM_PWD} -env GOROOT=${GOROOT} -env GOOS=wasip1 $bin -- -test.short -test.v"
```

Comment:
I _think_ GOROOT and GOOS won't be needed as envs here, but leave them for now

Reply:
GOOS probably not, but I am quite sure GOROOT is. Timezone tests fallback to look into GOROOT in some cases and on windows that pointed to c:\users\...\sdk\gotip which on the VFS of WASI would never make sense (I guess GOROOT defaults to the compile-time value?)

Reply:
(should have left it in a code comment...)